tests/unit/test_base_service.py

```diff
@@ -54,5 +81,207 @@ class Service(BaseService):
         'URI and Host both not matching',
     ]
 )
-def test_base_service_handles_request(base_service, request_, handled):
-    assert base_service.handles_request(request_) == handled
+def test_base_service_handles_request(service: BaseService, request_: dict, handled: bool):
+    assert service.handles_request(request_) == handled
+
+
+def test_base_service_get_responses_for_scenario(service: BaseService, scenarios_dir):
+    assert service.get_responses_for_scenario('test_scenario1') == [
```

Comment:
Save each of the sides in a variable before the comparison, for easy debugging in case of failure.